### DISCUSSION TITLE
Generating Large JSON Output Beyond Token Limit

### DISCUSSION THREAD

**Discussion by mainpyp** [openai/openai-python#1544] [Q&A]
Hello everyone,

I’m working on generating a sizable JSON output, often exceeding the maximum token limit of 4096. I’ve noticed that the generation process stops once the token limit is reached. Despite specifying the response format as `{ type: "json_object" }`, the generation is interrupted.

Is there a straightforward method to resume generation exactly where it left off?

I appreciate any insights or solutions you can provide.

Thank you in advance!

Feel free to adjust any details as necessary.

**Comment by ml-leviatan** [comment]
Hello
Use Openai Assistant API it can generate JSON output with more then 4093 tokens. I got JSON output swith 4770, 4590 and 4300.
